form/form.go

```diff
@@ -541,19 +540,11 @@ func (f *Values) Add(key, val string) {
 // Encode encodes the keys and values into “URL encoded” form
 // ("bar=baz&foo=quux").
 func (f *Values) Encode() string {
-	var buf bytes.Buffer
```

Comment:
Do you know where the indeterminism is coming from in the existing implementation? It looks like it should've been deterministic as is.

Reply:
When values are added using `AddTo`, and those values are maps, they are read in random order (as per go spec).

Reply:
I see, thank you for the explanation.